components/net/fetch/methods.rs

```diff
@@ -98,26 +99,27 @@ impl CancellationListener {
 pub type DoneChannel = Option<(TokioSender<Data>, TokioReceiver<Data>)>;
 
 /// [Fetch](https://fetch.spec.whatwg.org#concept-fetch)
-pub async fn fetch(request: &mut Request, target: Target<'_>, context: &FetchContext) {
+pub async fn fetch(fetch_params: &mut FetchParams, target: Target<'_>, context: &FetchContext) {
```

Comment:
I'm confused. The `fetch` algorithm receives a `Request`, not `FetchParams` according to the [specification](https://fetch.spec.whatwg.org/#concept-fetch)